data/language/en-GB.txt

```diff
@@ -4464,6 +4464,7 @@ STR_6152    :Invalid response from master server (no JSON array)
 STR_6153    :Pay to enter park / Pay per ride
 STR_6154    :It is not recommended to run OpenRCT2 with elevated permissions.
 STR_6155    :Neither KDialog nor Zenity are installed. Please install one, or configure from the command line.
+STR_6156    :Console
```

Comment:
This ID is reserved, since trigger-death accidentally opened an issue on Localisation for it already (and it's already translated in some languages).

Reply:
Fair enough. I'll happily rebase when his PR is merged, which hopefully won't be long.

Reply:
Your PR will be merged first, I think. Just change it.